apps/api/src/donations/donations.service.ts

```diff
@@ -152,14 +152,18 @@ export class DonationsService {
     }
   }
 
-  async getDonationsByUser(personId: string) {
+  async getDonationsByUser(personSub: string) {
     const donations = await this.prisma.donation.findMany({
       include: {
         targetVault: {
           include: { campaign: true },
         },
       },
-      where: { personId },
+      where: {
+        person: {
+          keycloakId: personSub,
```

Comment:
let's name the input parameter keycloakId instead of personSub, so that the where clause to look properly comparing same things. Also this will avoid the strangely named sub field from keycloak to propagate to our code ;)

Reply:
Done in https://github.com/podkrepi-bg/api/pull/214/commits/262da95dc6870e718662d88742a07fe9b821cb89